src/endpoint/s3/s3_bucket_logging.js

```diff
@@ -34,6 +53,15 @@ function endpoint_bucket_op_logs(op_name, req, res, source_bucket) {
     const s3_log = get_bucket_log_record(op_name, source_bucket, req, res);
     dbg.log1("Bucket operation logs = ", s3_log);
 
+    const buffer = Buffer.from(JSON.stringify(s3_log));
+    const client = create_syslog_udp_socket();
+    const url = new URL(req.object_sdk.rpc_client.rpc.router.syslog);
```

Comment:
Can we make this url parsing and port parsing in line below also one-time instead of for each request?

Reply:
will do that.